src/modules/dashboard/sagas/motions/paymentMotion.ts

```diff
@@ -24,26 +19,22 @@ import {
   transactionAddParams,
 } from '../../../core/actionCreators';
 
-function* getExtensionPermissionProofs(
-  // client: ExtendedIColony,
-  client: any,
+const getExtensionPermissionProofs = async (
+  colonyClient: any,
   domainId: BigNumberish,
   address?: string,
-) {
-  const [fundingPDID, fundingCSI] = yield call(
-    getPermissionProofs,
-    client,
+): Promise<[BigNumberish, BigNumberish]> => {
+  const [fundingPDID, fundingCSI] = await getPermissionProofs(
+    colonyClient,
     domainId,
     ColonyRole.Funding,
-    // address,
+    address,
   );
-
-  const [adminPDID, adminCSI] = yield call(
-    getPermissionProofs,
-    client,
+  const [adminPDID, adminCSI] = await getPermissionProofs(
+    colonyClient,
```

Comment:
I'll put this on my list for tommorow to make a new build of colonyJS that exports this for you.

_(I keep putting it off, as more "pressing" stuff always seems to take it's place. Sorry about that)_